src/components/availability-label/availability-label.tsx

```diff
@@ -45,7 +46,7 @@ export const AvailabilityLabel: React.FC<AvailabilityLabelProps> = ({
     accessTypes,
     access,
     faustIds,
-    isbn: isbns ? isbns[0] : null,
+    isbn: first(isbns) ?? null,
```

Comment:
I would maybe consider using the `||` operator here to make sure we catch all possible false-y values